models/project/column.go

```diff
@@ -58,17 +58,17 @@ func (Column) TableName() string {
 }
 
 // NumIssues return counter of all issues assigned to the column
-func (c *Column) NumIssues(ctx context.Context) int {
+func (c *Column) NumIssues(ctx context.Context) (int64, error) {
```

Comment:
The function is used by `template/projects/view.tmpl`, you need also to change that place.